src/Components/CreateImageWizard/steps/msAzure.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import componentTypes from '@data-driven-forms/react-form-renderer/component-types';
+import validatorTypes from '@data-driven-forms/react-form-renderer/validator-types';
+import { Title, Text, Button } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+export default {
+    title: 'Microsoft Azure',
+    customTitle: <Title headingLevel="h1" size="xl">Target Environment - Microsoft Azure</Title>,
+    name: 'ms-azure-target-env',
+    substepOf: 'Target environment',
+    nextStep: ({ values }) => values?.release === 'rhel-8' ? 'registration' : 'packages',
+    fields: [
+        {
+            component: componentTypes.PLAIN_TEXT,
+            name: 'azure-text-component',
+            label: <>
+                <Text>
+            Image Builder will send an image to an authorized Azure account.
+                </Text>
+                <Title headingLevel="h3">OAuth permissions</Title>
+                <Text>
+            To authorize Image Builder to push images to Microsoft Azure, the account owner
+            must configure Image Builder as an authorized application and give it the role of
+            &quot;Contributor&quot; to at least one resource group.<br />
+                </Text>
+                <Button
+                    component="a"
+                    target="_blank"
+                    variant="link"
+                    icon={ <ExternalLinkAltIcon /> }
+                    iconPosition="right"
+                    isInline
+                    href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow">
+                    <small>Learn more about OAuth 2.0</small>
+                </Button>
```

Comment:
This displays strangely and can be fixed by wrapping the whole button in small and not just the text.

```suggestion
                <small>
                  <Button
                      component="a"
                      target="_blank"
                      variant="link"
                      icon={ <ExternalLinkAltIcon /> }
                      iconPosition="right"
                      isInline
                      href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow">
                      Learn more about OAuth 2.0
                  </Button>
                </small>
```

![oauthlinkbroken](https://user-images.githubusercontent.com/11712857/122961876-1e749500-d385-11eb-8b8f-929d3e3ba786.png)
![oauthlinkfix](https://user-images.githubusercontent.com/11712857/122961908-27656680-d385-11eb-8fc6-811f34019f9f.png)